.github/workflows/ci.yaml

```
@@ -98,11 +98,4 @@ jobs:
             git add kubernetes/productcatalog/deploy.yaml
             git commit -m "[CI]: Update product catalog image tag"
-            git push origin HEAD:main -f
-
-        
-        
-          
-
-
-        
\ No newline at end of file
+            git push origin HEAD:main -f
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)